src/index.html

```diff
@@ -17,35 +16,35 @@
   </head>
   <body>
```

Comment:
Looks like you forgot to add class `page__body`.

Reply:
I did, thank you!